### PR TITLE
Upgrade for Laravel 8

### PR DESCRIPTION
**Pull request by codepotato**
Small composer.json tweak to allow installation for Laravel 8.

### CODE REVIEW

**Comment by codepotato** [comment]
I'm well aware that now the advice is to use Laravel Jetstream but there's no real reason why this cannot work on v8

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "keywords": ["laravel", "preset", "tailwindcss"],
   "license": "MIT",
   "require": {
-    "laravel/framework": "^7.0",
+    "laravel/framework": "^8.0",
```
Comment (innocenzi) [inline composer.json]:
```suggestion
    "laravel/framework": "^7.0|^8.0",
```

Comment (innocenzi) [reply composer.json]: Shouldn't it support both?

**Review by innocenzi — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (codepotato) [reply composer.json]: Yes, good point. When I did this earlier I clearly didn't have enough coffee in me!

**Comment by GJPhilippi** [comment]
Following [the Laravel upgrade guide](https://laravel.com/docs/8.x/upgrade) to upgrade to Laravel 8, it has a dependency of `laravel/ui` of version `"^3.0"`, when I now try to upgrade, it conflicts, as this package is still depending on version `"^2.0"` and doesn't accept version `"^3.0"`.

**Comment by driesvints** [comment]
@GJPhilippi laravel/ui v3 was only released 22 hours ago. Give the maintainers here some time to update.

**Comment by michaeldyrynda** [comment]
The upgrade guide, and `laravel/ui:^3.0` - were tagged _after_ we added Laravel 8.x support.

[4.3.1](https://github.com/laravel-frontend-presets/tailwindcss/releases/tag/4.3.1) has been updated to account for this now.

**Pull request merged**